web/src/drafts.js

```diff
@@ -689,12 +689,12 @@ export function launch() {
     $("#draft_overlay").css("opacity");
 
     open_overlay();
-    const first_element_id = [...formatted_narrow_drafts, ...formatted_other_drafts][0].draft_id;
+    const first_element_id = [...formatted_narrow_drafts, ...formatted_other_drafts][0]?.draft_id;
```

Comment:
What's the story with this line? It seems like it might belong in the previous commit.

Reply:
Yeah, that definitely belongs to the first commit, where I tried to make `set_initial_element` reusable. The optional chaining was added after the failed test, and it seems reasonable since there may be cases where there are no drafts or first element.